src/tests/member.test.js

```diff
@@ -1 +1,86 @@
-test.skip('skip', () => {});
+/* eslint-disable no-underscore-dangle */
+import request from 'supertest';
+import app from '../app';
+import Member from '../models/member';
+import memberSeed from '../seeds/member';
+
+const mockMember = {
```

Comment:
Maybe you can change this for all the data directly taken from the seed, so if the seed changes, the test doesn't fail.

`firstName: `${memberSeed[0].firstName}`,
  lastName: `${memberSeed[0].lastName}`,
  dni: `${memberSeed[0].dni}`,
  phone: `${memberSeed[0].phone}`,
  email: `${memberSeed[0].email}`,
  password: `${memberSeed[0].password}`,
  city: `${memberSeed[0].city}`,
  birthDay: `${memberSeed[0].birthDay}`,
  postalCode: `${memberSeed[0].postalCode}`,
  isActive: `${memberSeed[0].isActive}`,
  membership: `${memberSeed[0].membership}`,`
  
  Check if it's okay the sintaxis but i think it's okay

Reply:
Ok! I'm on it!..